test_api.py

```diff
@@ -1889,13 +1894,38 @@ def test_invalid_data(self):
 
                 self.verify_error_response(response, expected_title="Bad Request", expected_status=400)
 
-    def test_invalid_host_id(self):
+    def test_invalid_host_id(self, emit_event):
         patch_doc = {"display_name": "branch_id_test"}
         host_id_lists = ["notauuid", f"{self.added_hosts[0].id},notauuid"]
         for host_id_list in host_id_lists:
             with self.subTest(host_id_list=host_id_list):
                 self.patch(f"{HOST_URL}/{host_id_list}", patch_doc, 400)
 
+    def test_patch_produces_update_event(self, emit_event):
+        with patch("app.queue.egress.datetime", **{"now.return_value": self.now_timestamp}):
+            patch_doc = {"display_name": "patch_event_test"}
+            host_to_patch = self.added_hosts[0].id
+            self.patch(f"{HOST_URL}/{host_to_patch}", patch_doc, 200)
+            event_message = json.loads(emit_event.call_args[0][0])
+            emit_event.assert_called_once()
+            expected_event_message = {
+                "type": "updated",
+                "host": {
+                    "stale_timestamp": self.added_hosts[0].stale_timestamp.replace("T", " "),
+                    "display_name": "patch_event_test",
+                    "reporter": "test",
+                    "id": self.added_hosts[0].id,
+                    "account": "000501",
+                    "ansible_host": None,
+                    "tags": [{}],
+                },
+                "platform_metadata": {},
+                "metadata": {"request_id": "-1"},
+                "timestamp": self.now_timestamp.replace(tzinfo=timezone.utc).isoformat(),
```

Comment:
Just use `datetime.now(timezone.utc)` instead of `datetime.utcnow` in [setUp](https://github.com/RedHatInsights/insights-host-inventory/blob/a8c07151bd386b8d1c90bddafa13a42b55df3c71/app/queue/egress.py#L65) and you can save the replace.

```python
        self.now_timestamp = datetime.now(timezone.utc)
```

```suggestion
                "timestamp": self.now_timestamp.isoformat(),
```